src/OutputCacheProvider_net462/RedisOutputCacheProvider_net462.csproj

```diff
@@ -139,35 +65,33 @@
       <Link>ValueWrapper.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="packages.config" />
-  </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="..\OutputCacheProvider\RedisProviderResource.resx">
       <Link>RedisProviderResource.resx</Link>
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>RedisProviderResource.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <Import Project="..\..\build\Source.PostProcessing.targets" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.OutputCache.OutputCacheModuleAsync" Version="1.0.2" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="StackExchange.Redis" Version="2.5.61" />
+    <PackageReference Include="System.Diagnostics.PerformanceCounter" Version="6.0.1" />
+    <PackageReference Include="System.IO.Pipelines" Version="6.0.2" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
+    <PackageReference Include="System.Threading.Channels" Version="6.0.0" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="6.0.0" />
+    <PackageReference Include="Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers" Version="0.3.310801">
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>
   <PropertyGroup Condition="'$(TargetFrameworkVersion)' == 'v4.6.2'">
     <DefineConstants>$(DefineConstants);DOTNET_462</DefineConstants>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <OutputPath>$(OutputPath)$(TargetFrameworkVersion)\</OutputPath>
-    <Prefer32Bit>false</Prefer32Bit>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Can we remove this and let it generate assembly info?

Reply:
I tried removing the line and things broke: https://stackoverflow.com/questions/10311347/duplicate-assemblyversion-attribute

Reply:
Removing Properties\AssemblyInfo.cs should fix that